typings/index.d.ts

```diff
@@ -1328,6 +1328,7 @@ declare module 'discord.js' {
   }
 
   export class MessageComponentInteraction extends Interaction {
+    public componentType: MessageComponentType | MessageComponentTypes;
```

Comment:
This should always be the string, not integer:
```suggestion
    public componentType: MessageComponentType;
```
See:
https://github.com/discordjs/discord.js/blob/69bb5f911091e433ad3241d02b5beaca6426d433/src/structures/MessageComponentInteraction.js#L33
https://github.com/discordjs/discord.js/blob/69bb5f911091e433ad3241d02b5beaca6426d433/src/structures/MessageComponentInteraction.js#L60-L62

Reply:
Ah I saw that but forgot to change it, my bad!